bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -29,7 +30,12 @@
     BRIDGE_OFFLINE,
     FIRMWARE_UPDATING,
     DUTY_CYCLE,
-    BRIDGE_UNINITIALIZED;
+    BRIDGE_UNINITIALIZED,
+    /**
+     * Thing has been removed. Used for example when the thing has been removed from its bridge and the thing handler
+     * should be removed.
+     */
+    THING_GONE;
```

Comment:
You updated the docs, but here it should also be changed from THING_GONE to GONE, right ( + updating the JavaDoc comment)